coalib/bearlib/abstractions/Linter.py

```diff
@@ -467,6 +486,38 @@ def _create_config(cls, filename, file, **kwargs):
                     yield config_file
 
         def run(self, filename, file, **kwargs):
+            printer = ConsolePrinter()
+            log_printer = LogPrinter(NullPrinter())
+            sections, _ = load_configuration(None, log_printer)
+            local_bears, global_bears = collect_all_bears_from_sections(
+                sections, log_printer)
```

Comment:
this is really bad for performance, plus we already have the bears collected in coala. Maybe this check should rather happen in coala itself?

As pointed out on gitter, the linter is the wrong place, we'll want it for *all* bears.